plugins/circleci/src/index.ts

```diff
@@ -26,6 +26,7 @@ const envVars = {
 
 function pluginInit() {
   if (process.env.CIRCLECI) {
+    // eslint-disable-next-line no-console
     console.log(`writing circle ci environment variables to state...`)
```

Comment:
**question**: why isn't this logged by winston?

Reply:
Because it's called during initialisation, so before we create a winston instance. I'll add a comment to explain